cv.tex

```diff
@@ -150,102 +147,70 @@ \subsection{Programming}
 
 \section{Experience}
 
-\runsubsection{Coursera}
-\descript{| KPCB Fellow + Software Engineering Intern}
+\runsubsection{Dropbox}
+\descript{Sr. SecEng (IC4), Detections \& Response Team}
 
-\location{Expected June 2014 – Sep 2014 | Mountain View, CA}
+\location{Nov 2022 – Nov 2024 | Seattle, WA}
 \vspace{\topsep} % Hacky fix for awkward extra vertical space
 \begin{tightitemize}
-\item 52 out of 2500 applicants chosen to be a KPCB Fellow 2014.
+\item Lead Incident Responder for the Detections and Response Team (DART), seeing two high profile public security events \textbf{\href{https://dropbox.tech/security/changing-how-we-identify-malicious-urls-in-shared-documents}{[1]}} \textbf{\href{https://sign.dropbox.com/blog/a-recent-security-incident-involving-dropbox-sign}{[2]}}
+\item Transitioned security case management operations and metrics from Jira to Tines
+\item Co-developed and monitored insider risk detections for multiple Reduction in Force (RIF) events
+\item Lead emergent issues with Audit, Compliance, Risk teams, documenting playbooks and interaction outcomes
+\item Mentored several great security engineers into their first enterprise incident response role
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
 
-\runsubsection{Google}
-\descript{| Software Engineering Intern}
+\runsubsection{Amazon.com}
+\descript{Sr. SecEng (L5-6), Detections \& Monitoring Team}
 
-\location{May 2013 – Aug 2013 | Mountain View, CA}
+\location{July 2018 – June 2022 | Seattle, WA}
 \begin{tightitemize}
-\item Worked on the YouTube Captions team in primarily vanilla Javascript and Python to plan, design and develop the full stack implementation of a new framework to add and edit Automatic Speech Recognition captions.
-\item Created a backbone.js-like framework for the Captions editor.
-\item All code was reviewed, perfected, and pushed to production.
+\item Founded a team producing security alarms from GuardDuty, CloudTrail, and Linux endpoint telemetry sources
+\item Built, operated, and deprecated a multi-tenant cloud-native detection engine identifying security events from hundreds of thousands of AWS accounts
+\item Standardized alarm documentation through the \textbf{\href{https://blog.palantir.com/alerting-and-detection-strategy-framework-52dc33722df2}{Alerting and Detection Strategy (ADS) Framework}}
+\item Integrated our detections service into the Threat Intelligence Platform to communicate Mitre ATT\&CK coverage
+\item 2021 \textbf{\href{https://en.wikipedia.org/wiki/Log4Shell}{Log4Shell}} and \textbf{\href{https://www.wired.com/story/devastating-twitch-hack-sends-streamers-reeling/}{Twitch}} Large-Scale Security Event detections workstream lead
```

Comment:
(HG):
this may not be particularly clear to the reader. I may change this to a "compromise security incident" and align if you want with what Twitch publicly disclosed about it